packages/reader-syntax-trees-mode/src/ModeToolbar.vue

```diff
@@ -12,7 +12,12 @@
     props: ['expandAll'],
     methods: {
       onShow() {
-        this.$emit('show', !this.expandAll);
+        this.$emit(
```

Comment:
@paltman for consistency, what if we referenced these via a widget-level `constants` module?